src/scss/global/_core.scss

```diff
@@ -66,6 +66,10 @@ body {
   }
 }
 
+.hide-scrollbars * ::-webkit-scrollbar {
+  display: none;
+}
+
```

Comment:
to work with Firefox, it would be great to add 
```
.hide-scrollbars {
    scrollbar-width: none;
}
```
https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width

Reply:
Agreed and updated